nvme-wrap.h

```diff
@@ -24,6 +24,8 @@ int nvme_cli_identify_active_ns_list(struct nvme_dev *dev, __u32 nsid,
 				     struct nvme_ns_list *list);
 int nvme_cli_identify_allocated_ns_list(struct nvme_dev *dev, __u32 nsid,
 					struct nvme_ns_list *list);
+int nvme_cli_identify_secondary_ctrl_list(struct nvme_dev *dev, __u32 nsid,
+										  __u16 ctrl_id, struct nvme_secondary_ctrl_list *list);
```

Comment:
Some strange spacing happening here (and the next chunk). Do you have your tabs set to something other than 8 columns maybe?